addon/templates/components/frost-object-browser-inline.hbs

```diff
@@ -24,7 +24,11 @@
     {{/if}}
   <div class="content">
     <div class="tools">
-      <div class="sort"/>
+      <div class="sort">
+        {{#if sortData}}
+          {{frost-sort sortableProperties=sortData.sortableProperties sortParams=sortData.sortParams onChange=onSortChange}}
```

Comment:
Why not just expose both `sortableProperties` and `sortParams` on the object browser API?